dvc/command/plots.py

```diff
@@ -134,8 +134,8 @@ def add_parser(subparsers, parent_parser):
     plots_show_parser.set_defaults(func=CmdPlotsShow)
 
     PLOTS_DIFF_HELP = (
-        "Plot differences in metrics between commits in the DVC "
-        "repository, or between the last commit and the workspace."
+        "Show multiple versions of plot metrics "
+        "by plotting them in a single image."
```

Comment:
It's not always single image (confusion matrix requires to plot few of them). I think `Show multiple versions of plot metircs.` Would be fine

Reply:
Shame we lost the info about "between commits in the DVC repository, or between the last commit and the workspace." :slightly_frowning_face:

Reply:
What about "Show multiple versions of plot metrics from different commits"?

Reply:
Or we should go back to "Plot differences in metrics between commits in the DVC repository, or between the last commit and the workspace."

Reply:
Ok, I guess I caused some confusion here, by reviewing even the base version. @nik123 I am sorry about that. @jorgeorpinel could you take look? In docs, we have:
> This command visualize difference between metrics among experiments in the repository history.

That sounds about right.

Reply:
> It's not always single image (confusion matrix requires to plot few...

Isn't it technically a single image with a few plots in it though? Se have this "single image" text in iterative/dvc.org/pull/1382 right now.

> Shame we lost the info about "between commits in the DVC repository

That was Dmitry's rephrasing I think. Maybe Ivan's. Probably friendlier this way for the cmd ref since we can further explain it in the description, but I guess it could be different here so it has more detail. Up to you.

> In docs, we have...

Yes, that was from the description but it just changed per some feedback left by @shcheklein. The latest version is actually 

> This command is a way to visualize the "difference" between metrics among
experiments in the <abbr>repository</abbr> history, by plotting multiple
versions of the metrics.

---

I think we're having some problems with circular reviews between core and docs here guys 😅  How do we solve this deadlock? We should probably discuss here first, decide, merge, and then move onto docs.

Reply:
p.s. notice that there's already iterative/dvc.org/issues/1414 to apply these changes to docs when this is merged. Thanks

Reply:
>  I think we're having some problems with circular reviews between core and docs here guys 😅 How do we solve this deadlock? We should probably discuss here first, decide, merge, and then move onto docs.

I agree about discussing it here and then applying changes to https://github.com/iterative/dvc.org/pull/1382. @jorgeorpinel I've caused this deadlock by creating this PR before iterative/dvc.org/pull/1382 was approved. Sorry about that. I should have probably just waited.

Reply:
@efiop not quite, even for confusion matrix one could argue that we are dealing with single image. Take a look at this [example](https://vega.github.io/editor/#/url/vega-lite/N4IgJAzgxgFgpgWwIYgFwhgF0wBwqgegIDc4BzJAOjIEtMYBXAI0poHsDp5kTykBaADZ04JACyUAVhDYA7EABoQAEySYUqUMSSCGcCGgDaoJFEwMdaEOYBOmQXEUgcNuMppm3V2-cdLXxFYAEgCiAIIAIiAAvgomZhaC3gx2Dk4ubh6YXug+af5wgeihkTFxIKbmlrkpvumu7p7Kyal+IAHB4VGx8VVJNa31mU0tdQVFICXd5ZWJo-nODVk51rULHcVdZb1zA2OLw9nNe+uFnaU9FQnVq4NKGY1H820bk1uXszd5bQ-Lx7f7V5TbZXPrPIaPFbfJxA94za79AELX4jE4vM6bC7wsFoiF-cHjc7THZfNY-JaopHoibAj4IgkHSH-aGEzHE0G7Kl4yks9oYt5YkmI3kop641kC9mfYVk7lirkS2nYzkiiny3mwwUc0l3Rn48V8mlwoUM0VQ2WK43amW6s3Mi2Gokg6WmtXm3WaqX0g12hme53ehV6nkO-10nFB30GsPKnX7KNBmMmn1u+0e-lK5OR1N+jNWl0pw7uwF5rUF7NFtMlo1lwOqyu5mteiP1pmNp3hlUOhMa0vNru2nPRvsBlvdoeJkeduPIie9pujgfxuehqexm3LhvoKBqGFrrOt-UgHeYPcL6cb2dblQ0WRsCBIFJnjvr13X5RsMjPtmLm4nuUrP+lq1hGQHBvKYFJtaVhgQmkH7tB267vcE7weer5IaeKHXmhL5ZrBqHIY6P4XjBRFwURUHlgROGUQh1HkYRWHEZKv6IjRbaYd+rGkVx2GccedHofhjG0cxVGBhxR64SRGGCcxFHifRkmiQJMk8XJUmUupmaIfJAH-Dp+YqQpTHcbpDGmWJ5nGaBqnSUJeF6VpEGObJIlWWpbkaR5BlkUpwnOfZ2neRZJl+XxLFhXZnkOQFTmWRF+nAf2f7Ba58XuUFsUhZlPnZUlRkgZyLmAaFtklelZV5dFlU5RlNnFWl9XVY1qXsVVhnlU1HUtV1NUVc1SVMDQNjNClbH+cNo3jVFg29UlH5fhNvHJeBKwjWNbWTeg7h3g+T78Uee33o+NjbatJ0HedR2UldZ0XXJ92Het-zPTdc09VY72Lbep0vRJEY-bd8rA597XfX912-ftD0rU9UNw69kOwwDylA4jL0JmDgOcmD2OYx9uM3PjE44+jeOEzD-1ExTJNUyDKzk4F5ak++DPgztN6ox9BM849WZswJzMJYGQvHRzxOIuLd2S3T0sc3zNMC3pMug3LLNi4rZMa6LGP84zb261lrPa+zBuc5dZvC8b+WmxbSvQ-DgvWxLFtSyjyuG57TuWwjDs6+78s+0jjtIx7u2u7LQea-rXvI5HMd65TAfm-HEfc-HYdo7HKdZ4H6fB4n+dp77Gdq0ztu1fTqc20nJta7XcUq-bJdeQNX3F77ikt43bfN87embbNCbD73EZj976CT37WYz6PM3j5y88TjPGcr9ea9FyAG8CVvuc3LvR778nh+L1PO-n7PQ9XwvW2D+WR+UifDcT7fq9X+v7+b5-29P-KL87aBn-htX+B9EQgP+IA6uEDv57zAafWB98E6X2QV-ZBd9ZroJHh-NBf84HHwQa-ZeBDn5EKAW-DBuCsH4KoT-PB4CrCQKYeQmBLC6HwIYYg9hOD6E0MYdPUhADWHzR4dNLhxCz4cMIRIihJDpFkNkWwwRCjhFKNESo3hnD+HcM0eInRkikFaJkQYuRUjjGKNMco1BFi1FWI0TYwq3UIZ6IvkVFxjiL7zl0Z4lB7iuZLUWp+JeJNgkX0CQ-MWYSUEROvqzaJ2NonlwSWTJJ29YmJOWnEqJWTMkhOlik98aSBEqEKcLYpPiMmpKyck3J1T8nfTKcdCphjGl1KKTU9JTS7otLMQU9p5TOklKqR0hpu1umg16dYkZgyxmlIGc0oZlSJlMymQ4mZiy5kbJ6Us1p4yFk7K2Sst6azO7zKCbsvpbSLlHIOZMy50zjnXMiUDJ5+zbk3JeXjN55yvmhLuash56yfmxNqZ87JryAUnKBWc7Z9yPluOcQEn5-jLo-O8Xs35KDoEOOYa4lixAACMXM8W+NeESklQjQGyIpatUlgDaVyXpeQxlc8qVQJZcSul7KxESlZTfVR1LTH8sfjy-F5KuVMrFWS-kIrgHSoZZKtlgqOU0qVQK2xQruJysoZq1Vwr1WipVbyglhr5XGvFbKs1ur9HautfIvVJqJWUotTKiYOqHW2r5fa8x1My5WpdY64c7qfVGPBc67lrr1Iet9eE3pMaw2Irygmk1eTvWBqShikAKb8U93TZGoNk4Q0ZvCbbHNvi02mpLX47y5bSUi3QHW6V0bQ2poLv64tBbM2rk7VK11oKA1dqTXa6t2daa9oPOOa8Wby2HhDOmCdek53qh7VYWdU6BIztbYWTdq7G3borLuhda6D3LuLKcRd5Yz1VgvSerm172z7vvRuo8W7n2DmnXu7Np6X3zurHe1aD7g0AbkkBotIHJ0fqPf+p9gHf0ruPbB0D8Hz3Ugg0ulDN60NIcg5uaDt6cMYag6+r967iOHPzch8jDVKO4avLM2jRG8Nu0LpeusmGpqMavRxru4dB1UeY9HVj6HuPUa1Vx9jYmjb12-e+wT6sZNkfk5XRTP6pMhxziJyTynoUjrg+py1bGxwGbWhGgT9GB5Vv0zp55VnzPklGRJ4zNnDNaecxZ3KemnogvjQeuFgKvOCx85c8t-ndNOe+VC2zZmgtRfeRF-54b+OxaS0ZyLqW3PpbjSFvzwXAuqzywl-pGXCPxLi1imLBXysDrS4l7L+Wysldk2i6rvnkWtZy+1proWQVlty+Vt9LWvV2ZS8Oortmx0NZyX6vjtXiteNI-1mbmnSvTYvoqrrpbVObZrcmpb63OU7cm+N+Lu2puQu7Yh5r3mBuLaO+22bmW6soMGzd5b46nvzZiX1rmpV+rnbqk4vbv3OqcZGwVMb4PEqQ8q9Ds7J21p5qh+FGHyWIfw+RzFIHAOhqo7m2DpHsOUcY6J1jvHn2CdmQR39sHpPAfk9W2TknaO4eE5Z8Ttn+PIrHcx-Tl7d3Vo0+A4zvnla6e4+xYdwXoPXMi4l2L9nTOefi4WvV6nMuKuK9Fw9lb13fIHbVSDvqGmPty9V-zq75aheZw7RT7nOvTd6-RwrrniOVyW4Pdbmrdu3eOd5zXfuQnbdm5N+9nHCsm5B8eyH3jWMHfh9D1t4TMebeh3jwjiu0nk9O9bt3dP-uI+B4U9n0LUdi-B5z33PPpdo+V7jtXuuJe-Nl5U03gJLes8V9L5H8vtfu9F9b135vPfB99+HwPzvY-28j8n7r-vDeWND+nxPp1WuA8L8sQn2PvN6kZ479FtfheN80YL4nmJbWrYz9X67zPtPD9n85z72-wu6952P6hrfqe49+5Vw-3fp-t9htf9ACGdX91809P0Pdl9399VP9n9TN78QDJdDdL8V9IpgCsUXc10AAme7H-WVXAobNXAlQgt7Yg8lUg0bc-ELSgqrbrWgxrcgggvAhjEglgzZPlBgtbag7VLgi7Jg91PgrLHgzg9gijNgogkQiQsgqQigsQ+FUQyQrAxtIQ57ZQ7NVQr7dQ4gTQibf-OQpQ-Q5gww-AwQ+QgLRQmQ7Q3Q07aw8w8LaQqguwkw1ggwqw93GDDQ+ws-NwpwqnRwugg3YVGwzA-PXwwIi3TwnQ7wl-aIlwyzcIxgyIgjLw+IqPOfEIsLHw4w9w6ySw-XZnMw6XY3dAnIgox-FQo3bHfI53fwxIjnOoso2ovIgI1nRooozSDXaNEI63boqo0AuIzokohAjo8o9onA-owoiY4o6o1oholo+opXcYyomYgYnoro2tdY4Yvo1YqYlYoY2YxY7XBYpotok40Y5o9uXgyYio1Ig4tYm45Yu4sYvha43YzBN4+45I7DZ4y4jguYpY14mos41w04+YxvLvLY+gx4oEgE447RT4l4hE4E8EkxRE1WffJAo48A7-CEvvTIzEr-R3QYl2K-LEsE+vCAvEjI6AqkxffE2k3E+kmk1AmA7Ii43POk9I4kgkskok9Ezkpk7kgUqvLk3vFk-2NA-klEykoU8UnkxknfGvCU0kqUhtX4wUpU6khU1ksU0fFUjEvk9Ukkw0tUstXks0xTC0tk8kjk0UuU-UnUyUm06UuEnErU5kp01Ul04060vU2fL000n080xUw4ikt-f0g-O02Uj0v9FIk0zUy7KIv0h02AmUiM1Mx9DU+02MuxEUmMsM6MjM3M1qdM90oA8M8soI-M4sisos2NZA4I0dahGshsj4ssxNRs1szs9st0nslsjsttWE7E-s4cys0c5Evsocyckc6ctEwc3NAcqcxcsc+sic+c5citJc2clcmc8cuczfBcrc1c6YvtQtBlEI5lGlS8hVFlG811C85sk8-Y5Vc8u8p8vctcg8vMo8q8psodLs386VGde8wtb3F8jVMPIC11ECj8z07s78j-aCsCkMgC5Xfc3chIjC480Er8zC8QncnCrCvCw9EjS3EIsDWC6zDzBDZMuTGipCzcyi0jCinjcDCC0TFzdi7M9zBzfDH4hM7TBirDBCndMiui6ivi8S+M1ikzKi+zJMmS+iqSuMgS2Sri+SujFS2ipSySxStS5S-S0S0i1S4y5i8iwyhbCyvSqyiShS2y3S+y74sytizSpjYSrMwS3ioyo88yuyrS6EmywCpitix8oK3swisDHYpy9CkisDX0yy2Q7C+K1CmKp4rypcDy0ouK0K986IAAXViGsDoDSHQCcGQBsAAGsrBXAzAnA4BZAoA2A9ovxNAQAAAPNAUAAAMxoDgEEH+BdCUEwAAE8cBHB0A7wEBbxLAlAZA7ArAkBoAGqWqnBMASqJroIiqRquqQBer+r-hfRhqxrNqpqZqkg5q2AFr0AlqoAVrbwsl1q6h0BfQiqmrBArrdqkAyAyBXAKBsgYI2AGBZBmJRrxqrAABHCwEGugNQGgUgMoPa0wOAU8Nq-agamqs4Y68Gyatgaa2QSwaIImoAA) What we have here is three plots on one image, thats why I believe Jorge is right, and I was wrong in the beginning.

Reply:
It seems we almost have an agreement. @jorgeorpinel  what are your thoughts about last proposed text?

Reply:
Well, I lost track of this one until now oops, sorry! The text you came up with is good 🙂